docs/ChangeLog-NEW.md

```diff
@@ -0,0 +1,17 @@
+b33a76bd release workflow: adding pr permission to all release workflows (#1174)
+411faf2b changes to support write permission to create PR for gitaction (#1170)
+7087c969 CSPL-2396: Helm 2.3 support  (#1169)
+dfc5f54e CSPL-2388 - Fix helm int tests (#1165)
+2888d33e fix for service template (#1126)
+64c45a18 CSPL-2341 - Enable private access links as endpoints for AWS client sessions (#1147)
+b6e5fac8 cleaned up (#1155)
+492d43f5 CSPL_2354 - Fix nightly integration test (#1150)
+00dcd039 CSPL-2304 - Modify smartstore documentation for repFactor in internal indexes (#1146)
+4b681d36 cspl-2331: documentation changes for OIDC support (#1136)
+df5e9cc6 Merge pull request #1137 from splunk/fix-operator-doc-for-install
+9e09afad add --server-side  --force-conflicts to the installation step for operator
+957afb3d CSPL-2179 QA test for PR https://github.com/splunk/splunk-operator/pull/1043 (#1128)
+3726384d cspl-2340: Workflow failing due to docker issues (#1133)
+e4a847fe CSPL-2250 - Improve code coverage (#1092)
+bfb5b062 Updated .env to provide splunk enterprise release image (#1123)
+ba88fd0f Merge pull request #1122 from splunk/main-to-dev
```

Comment:
This change needs to be pruned and also it needs be in `ChangeLog.md` file and we need to remove this file.